app/views/layouts/application.html.erb

```diff
@@ -1,36 +1,5 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title><%= content_for(:title) || "Find A Buying Solution" %></title>
-    <meta name="viewport" content="width=device-width,initial-scale=1">
-    <meta name="apple-mobile-web-app-capable" content="yes">
-    <meta name="mobile-web-app-capable" content="yes">
-    <%= csrf_meta_tags %>
-    <%= csp_meta_tag %>
-
-    <%= yield :header %>
-
-    <%# Enable PWA manifest for installable apps (make sure to enable in config/routes.rb too!) %>
-    <%#= tag.link rel: "manifest", href: pwa_manifest_path(format: :json) %>
-
-    <%# Includes all stylesheet files in app/assets/stylesheets %>
-    <%= stylesheet_link_tag :app %>
-    <%= stylesheet_link_tag "application" %>
-
-    <%= javascript_include_tag "application", defer: true %>
-  </head>
-
-  <body class="govuk-template__body">
-    <script>document.body.className += ' js-enabled' + ('noModule' in HTMLScriptElement.prototype ? ' govuk-frontend-supported' : '');</script>
-
-    <% if controller_name == "categories" and  action_name == "index"%>
-      <%= render "layouts/dfe_homepage_header" %>
-    <% else %>
-      <%= render "layouts/dfe_header" %>
-    <% end %>
-
-     <%= yield %>
-
-    <%= render "layouts/dfe_footer" %>
-  </body>
-</html>
+<%= render "shared/dfe_base" do %>
+  <%= render "shared/dfe_homepage_header" %>
+  <%= yield %>
+  <%= render "shared/dfe_footer" %>
+<% end %>
```

Comment:
I like this! It's very easy to see what differs between the layouts.

Is this layout actually used? I think `application` is a better name for the default layout than `other`.